covid-sim.vcxproj

```diff
@@ -130,9 +130,9 @@
     <ClInclude Include="src\Update.h" />
     <ClInclude Include="src\MicroCellPosition.hpp" />
     <ClInclude Include="src\Direction.hpp" />
-    <ClInclude Include="src\Geometry\BoundingBox.h" />
-    <ClInclude Include="src\Geometry\Size.h" />
-    <ClInclude Include="src\Geometry\Vector2.h" />
+    <ClInclude Include="include\Geometry\BoundingBox.h" />
+    <ClInclude Include="include\Geometry\Size.h" />
+    <ClInclude Include="include\Geometry\Vector2.h" />
     <ClInclude Include="src\Models\Cell.h" />
```

Comment:
Should these have lower-case G on geometry? (Not that Windows really cares...)

Reply:
Good point, pays to be consistent. I'll change.